src/python/setup.py

```diff
@@ -17,6 +17,16 @@
 NON_MINIMAL_LIST = ["tensorflow", "resampy"]
 
 
+# pkgs not needed for minimal pkg
+NON_MINIMAL_LIST = [
```

Comment:
Do you want to delete the previous definition of `NON_MINIMAL_LIST` (i.e. what's on line 17)?

Reply:
That's added by auto-merge. I will remove it.